lib/elixir/lib/enum.ex

```diff
@@ -3809,9 +3809,6 @@ defmodule Enum do
       iex> Enum.unzip([{:a, 1}, {:b, 2}, {:c, 3}])
       {[:a, :b, :c], [1, 2, 3]}
 
-      iex> Enum.unzip(%{a: 1, b: 2})
-      {[:a, :b], [1, 2]}
```

Comment:
I don't see a relevant test to be added, a one-size map would hide the fact that the keys are misordered, we already have a keyword above...